hazelcast/src/main/java/com/hazelcast/internal/metrics/impl/MetricsConfigHelper.java

```diff
@@ -51,37 +48,37 @@ private MetricsConfigHelper() {
      *
      * @param config The configuration
      */
-    public static void overrideMemberMetricsConfig(Config config) {
+    public static void overrideMemberMetricsConfig(Config config, ILogger logger) {
         MetricsConfig metricsConfig = config.getMetricsConfig();
         MetricsManagementCenterConfig managementCenterConfig = metricsConfig.getManagementCenterConfig();
         MetricsJmxConfig jmxConfig = metricsConfig.getJmxConfig();
 
         // MetricsConfig.enabled
         tryOverride(ClusterProperty.METRICS_ENABLED, config::getProperty,
                 prop -> metricsConfig.setEnabled(Boolean.parseBoolean(prop)),
-                () -> Boolean.toString(metricsConfig.isEnabled()), "MetricsConfig.enabled");
+            () -> Boolean.toString(metricsConfig.isEnabled()), "MetricsConfig.enabled", logger);
```

Comment:
My IDE has gone mad and I had to recreate the project. I'm fighting with the formatter since then :) I made an attempt to fix the indenting (I think we don't need the "double-indentation" here).